sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/SwitchSectionShouldNotHaveTooManyStatements_DefaultValue.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    public class Program
+    {
+        public Program()
+        {
+            switch (myVariable)
+            {
+                case 0:
+                    break;
+                case 1: // Noncompliant {{Reduce this 'switch/case' number of lines from 6 to at most 5, for example by extracting code into a method.}}
+//              ^^^^^^^
+                    Console.WriteLine("1");
+                    Console.WriteLine("2");
+                    Console.WriteLine("3");
+                    Console.WriteLine("4");
+                    Console.WriteLine("5");
+                    break;
+                case 2:
+                    {
```

Comment:
Do we count blocks as one line? If yes, this test should be negative.

Reply:
The statements collection here will return only one statement, the block but as you said this is about readability so I thought I should expand blocks hence this count. Shall I change this?